editor/src/core/property-controls/property-controls-local.ts

```diff
@@ -161,6 +195,10 @@ export function createModuleEvaluator(editor: EditorState): ModuleEvaluator {
         },
         resolvedParseSuccess,
       )
+      const absoluteFilenameOrPackage = isRight(filePathResolveResult)
```

Comment:
I believe this should work:
```suggestion
      const absoluteFilenameOrPackage = defaultEither(toImport, filePathResolveResult)
```